src/main/java/frc/robot/commands/DefaultArmCommand.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.ArmControlSubsystem;
+import frc.robot.subsystems.Input;
+
+@SuppressWarnings("java:S2696")
+
+public class DefaultArmCommand extends CommandBase {
+  
+  private ArmControlSubsystem _armControlSubsystem;
+
+  public DefaultArmCommand(ArmControlSubsystem armControlSubsystem) {
+    this._armControlSubsystem = armControlSubsystem;
+    addRequirements(armControlSubsystem);
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    handleInput();
+  }
+
+  void handleInput(){
```

Comment:
Why is this in a separate function instead of just being inline in the execute function above?